src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,369 @@
+---
+title: Newcomers' Guide
+layout: about-page
+description: A newcomers' guide to Quilt - information, questions and concerns
+---
+
+This document is intended to be an overview of Quilt, and a way to address common questions and concerns by newcomers
+to The Quilt Project, its ecosystem or its community spaces. If you've been linked here, please read this page in full,
+as it may answer some questions you've been thinking about.
+
+To start with, though, let's get the basics out of the way...
+
+# What is Quilt?
+
+Quilt is a new mod-loader project for Minecraft. As explained on [the main "About" page](/about/), it exists in order
+to address the issues that have cropped up in other mod-loader projects over time, building upon the solid technical
+foundations provided by The Fabric Project. It aims to excel in many areas -- not just when it comes to its
+technology, but also in its decentralised and community-centric approaches to governance, its attempts to avoid 
+user-hostile software design, and its inclusive, harm-reducing approach to community management.
+
+While it's true that some of this work will appear abstract to some types of Quilt user, we believe that if a job is
+worth doing, then it's worth doing right -- and we all strive to make the Minecraft modding sphere better through our
+work.
+
+# Why not just contribute to other mod-loaders?
+
+We aim for Quilt to be an alternative mod-loader for users, rather than directly competing with other mod-loader
+projects. We believe that multiple projects that take different approaches to solving the problems that writing 
+Minecraft mods presents will push the ecosystem forward in a more productive way than a single mod-loader monopoly,
+and we hope that our work inspires other mod-loaders and motivates them to improve themselves alongside us.
+
+We currently work directly with Forge on moderation-related issues, events, and some technical problems. We also
+contribute relevant fixes to them, for cases where we've forked their projects -- such as
+[QuiltFlower](https://github.com/quiltMC/quiltflower).
+
+# Questions and Concerns
+
+Below, you'll find a list of common questions and concerns. Because some answers are quite long, they've been 
+collapsed -- please click or tap on them to read our response. Additionally, it's entirely possible that we've 
+forgotten to cover something here -- we'd love to hear from you if you think of anything that needs adding!
+
+{% admonition %}
+{% details %}
+{% summary %}What does Quilt provide over other mod-loaders?{% endsummary %}
```

Comment:
This is a guide for newcomers, so it's not specifically developer-oriented. Not sure the wording change makes sense, especially since we call it a mod-loader all over the site.